src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/PageViewModel.kt

```diff
@@ -2,7 +2,7 @@ package nl.avisi.structurizr.site.generatr.site.model
 
 import nl.avisi.structurizr.site.generatr.site.GeneratorContext
 
-abstract class PageViewModel(protected val generatorContext: GeneratorContext) {
+abstract class PageViewModel(val generatorContext: GeneratorContext) {
```

Comment:
Unfortunately we need the generator context for reading the svg files from disk. We could also add a property like `imagesBasePath` here instead of making `generatorContext` public.